superset-frontend/src/components/Chart/DrillBy/DrillByModal.tsx

```diff
@@ -62,18 +72,20 @@ const ModalFooter = ({ formData, closeModal }: ModalFooterProps) => {
   }, [dashboardPageId, datasource_id, datasource_type, formData]);
   return (
     <>
-      <Button buttonStyle="secondary" buttonSize="small" onClick={noOp}>
-        <Link
-          css={css`
-            &:hover {
-              text-decoration: none;
-            }
-          `}
-          to={url}
-        >
-          {t('Edit chart')}
-        </Link>
-      </Button>
+      {showEditChart && (
```

Comment:
I think we should still show the button even in table mode, and open the drill by chart just like before (I mean, with original viz type, not table). @kasiazjc wdyt?

Reply:
definitely agree with @kgabryje. Button should stay and should open the chart from the "chart" tab, as "table" tab is more like an underlying data, not viz